test/simulator/asm/org/apache/cassandra/simulator/asm/MethodLogger.java

```diff
@@ -37,6 +37,8 @@
 import static org.apache.cassandra.simulator.asm.MethodLogger.Level.NONE;
 import static org.apache.cassandra.simulator.asm.MethodLogger.Level.valueOf;
 
+// checkstyle: suppress below 'blockSystemPropertyUsage'
```

Comment:
This is because `org.apache.cassandra.simulator.asm` has its own classpath for compilation, so there is no access to the `org.apache.cassandra.config` package, but the checkstyle runs over all the source code. The test system properties from the `MethodLogger` do not seem to be relevant to Cassandra itself.

Here is how the `asm` classes are compiled:
```
        <javac compiler="modern" debug="true" debuglevel="${debuglevel}"
               source="${ant.java.version}" target="${ant.java.version}"
               encoding="utf-8" destdir="${simulator-asm.build.classes}" includeantruntime="true">
            <src path="${simulator-asm.build.src}" />
            <classpath>
                <fileset dir="${test.lib}">
                     <include name="**/asm-*${asm.version}.jar" />
                </fileset>
                <fileset dir="${build.lib}">
                     <include name="**/asm-*${asm.version}.jar" />
                </fileset>
            </classpath>
        </javac>
```